content/lessons/chapter-3/outro-1.tsx

```diff
@@ -0,0 +1,35 @@
+'use client'
+
+import { useTranslations } from 'hooks'
+import { ChapterEnd } from 'ui'
+
+export const metadata = {
+  title: 'chapter_three.outro_one.title',
+  image: '/assets/images/chapter-3-finish.jpg',
+  theme: 'bg-transparent/20',
+  secondaryTheme: 'bg-[#362042]',
```

Comment:
Lets go ahead and change the secondaryTheme to this color `bg-[#3c5c6e]`,  found from a dropper within the picture

Reply:
Addressed in e9c7b5d0c9ba853b0efb8b13594477225a469bd1
Thanks for the suggestion!